assignments/assignment01.rb

```diff
@@ -68,7 +76,12 @@ def my_reverse(s)
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
 def palindrome?(s)
-  # your implementation here
+    s = s.downcase
+    if s.reverse.scan(/\w/) == s.scan(/\w/) 
```

Comment:
whenever you want to return a boolean value and you have something that reads like so:

if test
  true
else
  false
end

then you don't need the if statement at all. You can simply use the test expression.